lib/components/viewers/viewers.css

```diff
@@ -219,24 +219,39 @@
 }
 
 .otp .stop-viewer .stop-viewer-body .next-arrival-row {
-  display: table;
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
+  align-items: center;
+  border-radius: 10px;
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 5px;
   width: 100%;
 }
+.otp .stop-viewer .stop-viewer-body .next-arrival-row:last-of-type {
+  margin-bottom: 0;
+}
 
 .otp .stop-viewer .stop-viewer-body .next-arrival-row .next-arrival-label {
-  display: table-cell;
-  max-width: 200px;
+  align-items: center;
+  display: flex;
+  max-width: 250px;
```

Comment:
max-width causes unnecessary clipping:

```suggestion
  flex-grow: 1;
  margin-right: 4px; /* or insert a non-shrinkable fixed-size element after. */
```

Reply:
I tried this originally but ran into the following issue on long destinations. Have tried all kinds of combos of flex shrinking and growing, and even grid but couldn't come up with something as clean as the max width that behaved appropriately. <img width="414" alt="Screen Shot 2022-11-28 at 8 54 10 AM" src="https://user-images.githubusercontent.com/86619099/204294828-04862e1a-cdd5-4a7a-a80a-ced4d43107d8.png">

Reply:
For this comment, I still think replacing `max-width` with `margin-right` renders better on wide screens.

Reply:
I agree, but this specific reflow issue to me is unacceptable